### DISCUSSION TITLE
gopro 12 x++, x--

### DISCUSSION THREAD

**Discussion by spiruline** [gopro/labs#715] [Feature Requests]
Hi David,
I have noticed that w++ w-- was fixed in one of the latest firmwares for GoPro 12.
How about x++ and x-- . Last time I checked it didn't work well on GoPro 12. Has this been solved with the latest firmware and hasn't been stated in the releases notes?
Best,

**Comment by dnewman-gpsw** [comment]
You want EV increment/decrement?  Should it wrap at -2/2 or at the Labs limits, which differs from the menus.  Doable either way.

**Comment by dnewman-gpsw** [comment]
Oh..  I did know x++ was implemented but broken, I get that fixed for the next release.  Congrats on the win, I will check it out.

**Comment by spiruline** [comment]


**Reply by dnewman-gpsw** [reply]
Yes that would be new firmware if it we are to change the default.  A simpler solution that works today:

![image](https://github.com/gopro/labs/assets/10856084/391cdb67-c2ef-47a2-9bc0-2f1020aa5ad6)

eL0 - auto expose (so we can update the lock)
eL3 - lock in 3 seconds
!B1 - Beep and Blink once

**Comment by spiruline** [comment]
Thank you !

That works well for gopro 11. But it doesn’t work for gopro 9: it changes to Photo mode after locking exposure.


Carlos Villoch

**Comment by spiruline** [comment]
!B1 : this is what makes the GoPro 9 to go into Photo mode



Carlos Villoch

**Reply by dnewman-gpsw** [reply]
Unfortunately it was not possible to be backwards compatible forever, and the are only 26 characters for action commands, there are a few newer commands that do different things on older cameras. You will need different codes for the older cameras.
